portfolio/src/main/webapp/index.html

```diff
@@ -16,6 +16,7 @@ <h1>Miah's Portfolio</h1>
     </div>
     <button id="button1"> </button>
     <button id="button2"> </button>
+    <a href="cs-page.html"><img src="/images/hair2.png" id="button2"></a>
```

Comment:
Always add alt text to images when you can https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback.

Also, let's try to give these id's more descriptive names (i.e. "cs-page-image-link").